user_config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "pools": {
+        "miningpoolhub": {
+            "api_key": "7f931dc99e1c794cb5e5b00850342e32cb3b37cdfe477eac81563abf1907c568"
+        }
+    }
+}
```

Comment:
Newline at end of file is absolutely critical python style

Reply:
Talked offline, .json file so yolo